site-scrapers/Curative.js

```diff
@@ -48,7 +48,7 @@ module.exports = async function GetAvailableAppointments(browser) {
             ).groups;
             const date = `${month}/${day}/${year}`;
             let newNumberAvailable = 0;
-            if (mappedData.availability[date]) {
+            if (mappedData.availability[date] && appointment.status != "Disabled") {
```

Comment:
Wouldn't it be better to change this test to the positive version...

`if (mappedData.availability[date] && appointment.status == "Active") {`

Reply:
or (as @livgust points out above) using "==="

`if (mappedData.availability[date] && appointment.status === "Active") {`

Reply:
I was trying to be just restrictive enough where it would work. The way that it was before ignored all values for status. Now, we only care about Disabled status. They probably just have "Disabled" and "Active", but who knows. If it's really a boolean, I probably would have used a boolean or called it "Disabled" and "Enabled" or "Active" and "Inactive". This way, they could have a status of "Enabled" and we'd still let the appointments through. Such is the problem with reverse engineering.